public/static/locales/en/common.json

```diff
@@ -1,3 +1,3 @@
 {
-  "new-event-type-btn": "New event type"
+  "new-event-type-btn": "New Event Type"
```

Comment:
I believe this to be an accidental change?

Reply:
i actually dont mind the change

Reply:
![image](https://user-images.githubusercontent.com/1046695/135729233-1b8a883e-9b21-4a71-9088-4176359db3d2.png)
![image](https://user-images.githubusercontent.com/1046695/135729245-f94f7549-6584-4afe-b12f-90f453f8daf0.png)

All of our buttons currently use "Lowercase continuing words" vs. "Camelcase Wording" - to my subjective eye "New event type" looks better than "New Event Type" but the most important thing here is consistency.